llm/server.go

```diff
@@ -1545,14 +1581,16 @@ func (s *llmServer) Completion(ctx context.Context, req CompletionRequest, fn fu
 }
 
 type EmbeddingRequest struct {
-	Content string `json:"content"`
+	Content  string `json:"content"`
+	Truncate bool   `json:"truncate"`
```

Comment:
Similar to the comment below, I'm not sure we need this anymore as we will always error on truncate in the runner 😊